src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+input {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-bottom: 20px;
+  margin-top: 20px;
+  padding-left: 62px;
+  font-family: inherit;
+  font-weight: inherit;
+  font-size: inherit;
+  color: inherit;
+}
+
+input:hover {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  outline: none;
+}
+
+.search-form-big input {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  object-position: center;
+  background-position: 26px;
+}
+
+.search-form-small input {
+  height: 42px;
+  background-size: 14px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 13px 15px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+input::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
```

Comment:
still used tags

u can create class input and put in on both inputs

style class input as you need for big 

then add class `input-small` to small and rewrite some styles that you need